.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
I have added this .npmrc because I run with package-lock disabled. I hope you don't mind - lmk if you want this removed.

Reply:
I'm okay with this! I'm curious about running with package-lock disabled though. Is it because it doesn't matter for published packages? I run with them because its been advised but another part of me wants to ditch them too.

Reply:
https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514 this sums it up.